apps/src/javalab/Theater.js

```diff
@@ -94,6 +106,39 @@ export default class Theater {
   }
 
   onPhotoPrompterFileSelected(photo) {
-    // TODO: Send photo file information back to Javabuilder.
+    if (!this.prompterUploadUrl) {
+      // The upload URL should be provided when opening the prompter, so if
+      // it is somehow not set, we are in an invalid scenario.
+      console.warn('No upload URL available. Cannot upload prompter image.');
```

Comment:
do we still want a console warning here?

Reply:
Yeah, i thought a warning would be useful since this indicates some weird state - we should always be getting an upload URL if this is being called.

Reply:
is it useful to have it be to the web console though? Is this just for dev purposes? We will see a message to the javalab console anyway so we will know something went wrong.

Reply:
Hmm yeah that's true, I thought it might be useful for dev/debugging purposes but yeah we will see a Javalab console message as well. I'll remove this.